editor/components/writing-flow/index.js

```diff
@@ -205,6 +243,32 @@ class WritingFlow extends Component {
 		}
 	}
 
+	componentDidUpdate( prevProps ) {
+		// When selecting a new block, we focus its first editable or the container
+		if (
+			this.props.selectedBlock &&
+			( ! prevProps.selectedBlock || this.props.selectedBlock.uid !== prevProps.selectedBlock.uid )
+		) {
+			const blockContainer = this.container.querySelector( `[data-block="${ this.props.selectedBlock.uid }"]` );
+			if ( ! blockContainer.contains( document.activeElement ) ) {
+				const target = this.getInnerTabbable( blockContainer, this.props.initialPosition === -1 );
+				target.focus();
+				if ( this.props.initialPosition === -1 ) {
```

Comment:
Minor: While browsing the code elsewhere, I failed to understand what `initialPosition` was. I'm guessing by initial position you mean, when manually focussing for the first time, set this as the offset. I'm also guessing that this can only ever be two values? `0` (potentially, not used) and `-1`? Could we make it boolean? Maybe similar to `isReverse` or something like `isCaretAtEnd`?

Reply:
yep, that's exactly the case :) I'm open to making it a boolean